api/db-cdr/generate-cdr/build-review-all-events.sh

```diff
@@ -6,7 +6,10 @@ set -e
 
 export BQ_PROJECT=$1   # project
 export BQ_DATASET=$2   # dataset
+export DATA_BROWSER=$3 # data browser flag
```

Comment:
This needs to be removed. The conditional workflows will take care of this.

Reply:
Good to know, done.